bot/bot.py

```diff
@@ -63,6 +63,7 @@ def load_extensions(self, exts: List[str]):
 
         await devlog.send(embed=embed)
 
+    @Cog.listener()
```

Comment:
```suggestion
```

Reply:
I thought I had double checked this to see if its removal made the cooldown reset not work. Will double check & apply suggestion when I get home from work tonight.

Reply:
I'll address the cog loading statements in #146

Reply:
@scragly is this resolved?

Reply:
No @jchristgit, as there's no follow-up or edit applied to the code.
In the time since the last review though I've double checked my original statement and I can confirm the usage of the listener deco on the bot method isn't correct.

However, the fact that there's an error handler on the bot class as a method is actually incorrect in the first place, and counters the project structure of handling global errors within the proper error handler in `/bot/seasons/evergreen/error_handler.py`. Because of this I'm simply adjusting the code so this is corrected entirely.